src/SeleniumLibrary/locators/elementfinder.py

```diff
@@ -119,11 +94,10 @@ def unregister(self, strategy_name):
         if strategy_name in self._default_strategies:
             raise RuntimeError("Cannot unregister the default strategy '%s'."
                                % strategy_name)
-        elif strategy_name not in self._strategies:
-            logger.info("Cannot unregister the non-registered strategy '%s'."
-                        % strategy_name)
-        else:
-            del self._strategies[strategy_name]
+        if strategy_name not in self._strategies:
+            raise RuntimeError("Cannot unregister the non-registered strategy '%s'."
```

Comment:
Should there be issue about this change? It by a quick look feels backwards incompatible change.

Reply:
I guess so. Submitted #961.